internal/controllers/clusterextension_controller.go

```diff
@@ -206,8 +206,7 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	installedBundle, err := r.InstalledBundleGetter.GetInstalledBundle(ctx, ext)
 	if err != nil {
 		setInstallStatus(ext, nil)
-		// TODO: use Installed=Unknown
-		setInstalledStatusConditionFailed(ext, err.Error())
+		setInstalledStatusConditionUnknown(ext, err.Error())
 		setStatusProgressing(ext, err)
```

Comment:
Nit-ish: should the progressing message end up being something like: "trying again to get installed bundle" rather than a repeat of the message in the Installed status condition?

Reply:
Sorry, I think the error message should go on the Installed condition and the "retrying to get install bundle" should go on the Progressing condition.

The Installed condition is unknown _because_ of that error.